metadome/presentation/web/routes.py

```diff
@@ -19,6 +19,10 @@
 def index():
     return render_template('index.html')
 
+@bp.route('/transcript/<transcript_id>/', methods=['GET'])
```

Comment:
Please make the lnk '/dashboard/transcript/<transcript_id>'

Reply:
Yes, that's possible.